style.css

```diff
@@ -88,21 +75,377 @@ h3 {
   line-height: 24px;
   color: #fefcfc;
 }
-/* stylelint-disable-next-line no-descending-specificity */
+
 .social-media a {
   padding: 1rem;
   color: white;
   font-size: 1.5rem;
   transition: 3s;
 }
 
+.nav a {
+  text-decoration: none;
+  color: #ffff;
+  padding: 0.7rem;
+  font-size: 1rem;
+}
+
 .social-media a:hover {
   text-shadow: #f87213;
   background: hwb(25 0% 0%);
   color: rgb(97, 91, 91);
   transition: 0.4s;
 }
 
-li {
-  display: inline;
+.nav a:hover {
+  background: hwb(25 0% 0%);
+  color: rgb(97, 91, 91);
+  transition: 0.4s;
+}
+
+.social {
+  display: flex;
+  justify-content: space-around;
+}
+
+.mainmyworks {
+  background-color: rgb(251, 249, 249);
+  padding-top: 40%;
+  margin-bottom: 20%;
+  padding-bottom: 40%;
+}
+
+.headtwo {
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: 'Crete Round';
+  font-style: normal;
+  font-weight: 200;
+  font-size: 30px;
+  line-height: 52px;
+  margin-bottom: 25px;
+  display: flex;
+  justify-items: center;
+  color: #010101;
+  order: 0;
+  flex-grow: 0;
+  text-decoration: none;
+  padding-left: 20%;
+  padding-right: 10%;
+}
+
+.liner {
+  color: rgb(11, 0, 0);
+  border-bottom: none;
+  margin-bottom: 80px;
+}
+
+.workitems {
+  display: grid;
+  grid-template-rows: 1fr;
+  grid-template-columns: 1fr;
+  padding: 10%;
+}
+
+.photo img {
+  width: 100%;
+  position: relative;
+  padding-top: 0%;
+}
+
+.headfour {
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: 'Crete Round';
+  font-style: normal;
+  font-weight: 200;
+  font-size: 25px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #01050c;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  padding-left: 7px;
+}
+
+.page p {
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: 'Inter';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #00040a;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  padding: 10px;
+}
+
+.pagelinks {
+  display: flex;
+  width: 90%;
+  justify-content: space-around;
+  align-items: center;
+  flex-wrap: wrap;
+  gap: 6px;
+  padding: 15px;
+}
+
+.pagelinks li {
+  display: flex;
+  justify-content: space-around;
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: 'Inter';
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 10px;
+  color: black;
+  border: #888b90 solid 0.1px;
+  width: fit-content;
+  padding: 8px;
+}
+
+.button {
+  /* background-color: #ff6b00; */
+  padding: 12px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 16px;
+  font-style: normal;
+  color: #fdfcfc;
+  line-height: 24px;
+  cursor: pointer;
+  border: none;
+  width: fit-content;
+}
```

Comment:
As suggested by the previous reviewer, please fix the padding on the button and the technologies. I suggest you make sure that they are aligned just as the figma design

- [ ] your design

<img width="362" alt="Screenshot 2022-09-26 at 13 13 21" src="https://user-images.githubusercontent.com/48546992/192253656-7077a9bf-2ad3-4472-b554-5ffd31d4a7eb.png">